python/lsst/ts/wep/instrument.py

```diff
@@ -104,6 +105,11 @@ class Instrument:
         detector offset is then used for everything else. Note that
         depending on the model, the sign of this value might matter.
         (the default is None)
+    heightMap : str or astropy.table.Table or None, optional
```

Comment:
Somewhere we should write down the units in addition to the dimensions.  (mm, micron, ?).
Also the coordinate system.